consensus/state.go

```diff
@@ -320,10 +320,7 @@ func (cs *ConsensusState) OnStop() {
 
 	cs.timeoutTicker.Stop()
 
-	// Make BaseService.Wait() wait until cs.wal.Wait()
```

Comment:
what is the motivation behind this change?

Reply:
SDK tests are failing non-deterministically with "cant open wal". Those tests do a lot of stopping and starting. So I went looking into this and noticed that the code here actually never ran and we never stopped the WAL - unless I'm missing something.